### PR TITLE
Parallelization of build_all.py

### PR DESCRIPTION
**Pull request by jjacobson95**
Everything is working. Ready to Merge.

- All Docker Images are now built in parallel using docker compose.
- Samples and Drugs file generation should now run in parallel. 
- All Omics and Experiments file generation should run in parallel once all samples and drugs are completed.
- CPTAC version updated to the latest ver. The 1.5.8 version was not working for me, so I updated to 1.5.11 and it worked - although sounds like this may not work for you, based on [cptac issue #60](https://github.com/PayneLab/cptac/issues/60)?
- Push to Figshare is working. Required arguments: --figshare --version
Push to PyPI is working. Required arguments:  --pypy --version
- The version argument expects a value such as 0.1.28.
- high_mem argument for full parallelization. Exclude it for semi-parallelization.
- Schema checker integrated and each dataset runs in parallel.
- [README.md](https://github.com/PNNL-CompBio/coderdata/blob/docker-build-multi/build/README.md) file updated

In summary, this may be run on a high memory platform, fully validate, upload to figshare and and upload to pypi with the following commands:
```
export SYNAPSE_AUTH_TOKEN="..."
export PYPI_TOKEN="..."
export FIGSHARE_TOKEN="..."

python build/build_all.py --all --high_mem --pypi --figshare --version 0.1.29
```

Edit - 
Also manually updated all files needed to update github pages which is now live.

### CODE REVIEW

**Comment by jjacobson95** [comment]
@sgosline This should be ready to go.  Full with with high_mem (and without) was successful.  Latest HCMI changes haven't been tested yet, but that shouldn't impact this PR specifically.

This took ~16 hours from start to finish on a c5.9xlarge EC2 instance (36vCPUs, 72Gb Memory).

**Comment by jjacobson95** [comment]
One note, the [build/docker/Dockerfile.upload](https://github.com/PNNL-CompBio/coderdata/blob/docker-build-multi/build/docker/Dockerfile.upload#L11) file will have to be updated to remove this line after testing is complete.
`RUN git checkout docker-build-multi`
